src/screens/orderreport/view/OrderReport.js

```diff
@@ -24,34 +19,19 @@ export default function OrderReport({ route, navigation }) {
   const [startDate, setStartDate] = useState(oneMonthAgo());
   const [endDate, setEndDate] = useState(new Date());
   const [flag, setFlag] = useState(false);
-  const[products,setProducts]=useState([])
+  const [products, setProducts] = useState([]);
 
   const date = new Date();
-  
-  
-  
-
-  
 
   const { user } = useAuthContext();
   let userId = user.userId;
 
-  
-
-  const data = [
-    { inm: "dosa batterr", Qty: 20, price: 200 },
-    { inm: "Chapati", Qty: 30, price: 300 },
-    { inm: "Ata", Qty: 20, price: 200 },
-  ];
-
-  useEffect(()=>{
-    getOrderReport(userId,date).then((res)=>{
-      console.log(res.data);
+  useEffect(() => {
+    getOrderReport(userId, date).then((res) => {
       setProducts(res.data);
-    })
-  },[userId,date]);
+    });
+  }, [userId, date]);
```

Comment:
Date should be a state variable, bound to Date picker. Otherwise it wont work as expected